Fabric.Authorization.API/Infrastructure/PipelineHooks/OnErrorHooks.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Fabric.Authorization.API.Constants;
+using Fabric.Authorization.API.Models;
+using Nancy;
+using Nancy.Responses.Negotiation;
+using Serilog;
+
+namespace Fabric.Authorization.API.Infrastructure.PipelineHooks
+{
+    public class OnErrorHooks
+    {
+        private readonly ILogger _logger;
+
+        public OnErrorHooks(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        internal dynamic HandleInternalServerError(NancyContext context, Exception exception,

```

Comment:
Should we interrogate the environment variable that represents if this is a production vs debug environment, and if its a debug environment, return the stack trace in the error response?

Reply:
@michaelvidal24 i made the change to return the stack trace and the real exception message when running in development mode